test/e2e/scaffold/k8s.go

```diff
@@ -184,6 +184,30 @@ func (s *Scaffold) EnsureNumApisixUpstreamsCreated(desired int) error {
 	return ensureNumApisixCRDsCreated(u.String(), desired)
 }
 
+// GetServerInfo collect server info from "/v1/server_info" (Control API) exposed by server-info plugin
+func (s *Scaffold) GetServerInfo() (map[string]interface{}, error) {
+	u := url.URL{
+		Scheme: "http",
+		Host:   s.apisixControlTunnel.Endpoint(),
+		Path:   "/v1/server_info",
+	}
+	resp, err := http.Get(u.String())
```

Comment:
deferred to close the body.

Reply:
I added a `defer` statement if no error is returned.